client/my-sites/checkout/composite-checkout/hooks/product-variants.tsx

```diff
@@ -207,7 +207,10 @@ function VariantPrice( { variant }: { variant: AvailableProductVariantAndCompare
 		variant.introductoryOfferPrice !== null
 			? variant.introductoryOfferPrice
 			: variant.priceFinal || variant.priceFull;
-	const isDiscounted = currentPrice !== variant.priceFullBeforeDiscount;
+	// extremely low "discounts" are possible if the price of the longer term has been rounded
+	// but if the discount is only a few percentage points we should not really call attention to it.
+	const isDiscounted =
+		( variant.priceFullBeforeDiscount - currentPrice ) / variant.priceFullBeforeDiscount >= 0.05;
```

Comment:
This looks great to me, although I wonder: how did you chose `0.05` as the threshold? It feels a little like a "magic number" to me in this context. What if instead we used the same logic `VariantPriceDiscount` uses for displaying the percentage, and hid the number if it would be 0?

Reply:
5% is indeed a magic number, based on what I imagined people may be irritated seeing ( 1% doesn't feel very meaningful, for example ). However, I think that using the same logic is the way to go.